.vscode/settings.json

```diff
@@ -0,0 +1,30 @@
+{
+    "cmake.generator": "Ninja",
+    "cmake.configureArgs": [
+        "-DCMAKE_INSTALL_PREFIX=${workspaceFolder}/build/install",
```

Comment:
while we are at it, would you mind adding CMakeUserPresets.json?

 see https://github.com/audacity/audacity-private-docs/blob/main/build/CMake%20Presets.md for AU3
 
 Or we could do that within another PR

Reply:
You see, I missed that. Will consider it for this PR.